GitUI/CommandsDialogs/FormRebase.Designer.cs

```diff
@@ -264,13 +270,39 @@ private void InitializeComponent()
             this.chkStash.Anchor = System.Windows.Forms.AnchorStyles.Left;
             this.chkStash.AutoSize = true;
             this.chkStash.Enabled = false;
-            this.chkStash.Location = new System.Drawing.Point(343, 3);
+            this.chkStash.Location = new System.Drawing.Point(325, 3);
             this.chkStash.Name = "chkStash";
-            this.chkStash.Size = new System.Drawing.Size(54, 19);
+            this.chkStash.Size = new System.Drawing.Size(76, 17);
             this.chkStash.TabIndex = 8;
             this.chkStash.Text = "Auto stash";
             this.chkStash.UseVisualStyleBackColor = true;
             // 
+            // chkIgnoreDate
+            // 
+            this.chkIgnoreDate.Anchor = System.Windows.Forms.AnchorStyles.Left;
+            this.chkIgnoreDate.AutoSize = true;
+            this.chkIgnoreDate.Location = new System.Drawing.Point(407, 3);
+            this.chkIgnoreDate.Name = "chkIgnoreDate";
+            this.chkIgnoreDate.Size = new System.Drawing.Size(80, 17);
+            this.chkIgnoreDate.TabIndex = 9;
+            this.chkIgnoreDate.Text = "Ignore date";
+            this.toolTip1.SetToolTip(this.chkIgnoreDate, "Changes the author dates of the commits in this branch\r\ninto the commit dates.");
+            this.chkIgnoreDate.UseVisualStyleBackColor = true;
+            this.chkIgnoreDate.Click += new System.EventHandler(this.chkIgnoreDate_Click);
+            // 
+            // chkCommitterDateIsAuthorDate
+            // 
+            this.chkCommitterDateIsAuthorDate.Anchor = System.Windows.Forms.AnchorStyles.Left;
+            this.chkCommitterDateIsAuthorDate.AutoSize = true;
+            this.chkCommitterDateIsAuthorDate.Location = new System.Drawing.Point(493, 3);
+            this.chkCommitterDateIsAuthorDate.Name = "chkCommitterDateIsAuthorDate";
+            this.chkCommitterDateIsAuthorDate.Size = new System.Drawing.Size(163, 17);
+            this.chkCommitterDateIsAuthorDate.TabIndex = 10;
+            this.chkCommitterDateIsAuthorDate.Text = "Committer date is author date";
+            this.toolTip1.SetToolTip(this.chkCommitterDateIsAuthorDate, "Changes the committer dates of the commits in this branch\r\ninto the author dates.");
+            this.chkCommitterDateIsAuthorDate.UseVisualStyleBackColor = true;
+            this.chkCommitterDateIsAuthorDate.Click += new System.EventHandler(this.chkCommitterDateIsAuthorDate_Click);
```

Comment:
Apologies, I totally missed it.
The correct event here is [`CheckedChanged`](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.checkbox.checkedchanged?view=netframework-4.6.1)

Reply:
Yeah it seems like all the check-boxes on the form should have been using `CheckedChanged` events in the first place. I'll change the one for `chkInteractive` along with changing the ones for `chkIgnoreDate` and `chkCommitterDateIsAuthorDate`. If nothing else, it will simplify the test suite.